collectors/gsuite/README.md

```diff
@@ -10,35 +10,57 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/1219251?hl=en&ref_topic=4514341)
+2. This admin role should have ‘Reports’ privilege.
+3. Create an admin user and assign him the role created above.
+4. Enable API access for G Suite. [Link](https://support.google.com/a/answer/60757?authuser=3)
+
+### 2. Google Cloud platform Setup
+
+1. Create a google cloud project [Link](https://console.cloud.google.com/home/dashboard)<br />
```

Comment:
Should say here how to create a Google Cloud project
`Click the top left dropdown menu, then New Project`